tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
+1 on that.

`name` shouldn't be an instance attribute. It doesn't change in any of the cases.